packages/plugins/Debugger/src/SNSAdaptor/components/ConsoleContent.tsx

```diff
@@ -86,16 +90,33 @@ export function ConsoleContent(props: ConsoleContentProps) {
             content: <Typography variant="body2">{blockTimestamp}</Typography>,
         },
         {
-            name: 'Token List',
+            name: 'Lookup Address',
             content: (
-                <Button
-                    size="small"
-                    onClick={async () => {
-                        const token = await onSelectFungibleToken()
-                        console.log(token)
-                    }}>
-                    Select Fungible Token
-                </Button>
+                <Typography variant="body2">
+                    <span>{lookedAddress}</span>
+                    <Button
+                        size="small"
+                        onClick={() => {
+                            retryLookedAddress()
+                        }}>
+                        Lookup Address
+                    </Button>
+                </Typography>
+            ),
+        },
+        {
+            name: 'Reversed Name',
+            content: (
+                <Typography variant="body2">
+                    <span>{reversedName}</span>
```

Comment:
Add a `<input />` for custom ens.

Reply:
It's a read-only field for test purposes.